apm-protocol/apm-ui-protocol/src/main/resources/ui-graphql/common.graphqls

```diff
@@ -0,0 +1,100 @@
+schema {
+    query: Query
+}
+
+#Root node
+type Query {
+    version: String
+}
+
+# The Duration defines the start and end time for each query operation.
+# Fields: `start` and `end`
+#   represents the time span. And each of them matches the step.
+#   ref https://www.ietf.org/rfc/rfc3339.txt
+#   The time formats are
+#       `SECOND` step: yyyy-MM-dd HHmmss
+#       `MINUTE` step: yyyy-MM-dd HHmm
+#       `HOUR` step: yyyy-MM-dd HH
+#       `DAY` step: yyyy-MM-dd
+#       `MONTH` step: yyyy-MM
+# Field: `step`
+#   represents the accurate time point.
+# e.g.
+#   if step==HOUR , start=2017-11-08 09, end=2017-11-08 19
```

Comment:
In the section  of 5.8. Examples of https://www.ietf.org/rfc/rfc3339.txt. 
```
Here are some examples of Internet date/time format.

      1985-04-12T23:20:50.52Z
```
Now, we did not support the time zone. So, 1985-04-12T23:20:50 is better than 2017-11-08 9:00.

Reply:
The specific says:
```
NOTE: ISO 8601 defines date and time separated by "T".
      Applications using this syntax may choose, for the sake of
      readability, to specify a full-date and full-time separated by
      (say) a space character.
```

I assume the ' '(whitespace) is OK.

Reply:
I prefer to use "T" for separated character

Reply:
In my perspective, **T** is not a tradition. We never use this char in Java application, when we need to format a time. So I want to keep it in that way. @peng-yongsheng argues with this, because he miss the `NOTE`.

So, let's keep the time format, OK?

Reply:
OK